src/Orchard.Web/Modules/Orchard.Azure.Authentication/Migrations/AzureActiveDirectorySyncPartMigrations.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Orchard.Data.Migration;
+using Orchard.Data.Migration.Schema;
+using Orchard.Environment.Extensions;
+using Orchard.Azure.Authentication.Models;
+using System;
+
+namespace Orchard.Azure.Authentication.Migrations
+{
+	public class AzureActiveDirectorySyncPartMigrations : DataMigrationImpl
+	{
+
+		public int Create()
+		{
+			SchemaBuilder.CreateTable(typeof(AzureActiveDirectorySyncPartRecord).Name,
```

Comment:
Correct, this is what we do in migrations, even with C# 6 we don't use `nameof`.